package.json

```diff
@@ -13,7 +13,7 @@
     "packages/*"
   ],
   "scripts": {
-    "build": "yarn ts-bridge --project tsconfig.build.json --verbose && yarn workspace @metamask/design-system-twrnc-preset build && yarn workspace @metamask/design-system-react-native build",
+    "build": "yarn ts-bridge --project tsconfig.build.json --verbose",
```

Comment:
So we're not building the other 2 packages?

Reply:
Apologies I was doing some testing and changed this PR to read for review when it is indeed not ready!  I couldn't work out what the criteria was for a publish build to be skipped. We would still be building these packages. I'll change this PR back to draft 

<img width="1497" alt="Screenshot 2024-11-26 at 12 00 40 PM" src="https://github.com/user-attachments/assets/364a2428-063d-41aa-8eca-f17cc92920e7">